docs/_sections/forms.html

```diff
@@ -65,6 +65,8 @@ <h3 id="requiredFields">Required Fields</h3>
 
     <p>Required fields must include a <code>span</code> tag of class <code>nypl-required-field</code> in the label that includes the text “Required”. The field itself should include the <code>required</code> and <code>aria-required</code> attributes.</p>
 
+    <p>It is recommended that JavaScript-based validation be made in the <code>onchange</code> event (native to <code>select</code>, <code>input[type=checkbox]</code>, and <code>input[type=radio]</code>, also fired by some frameworks on text <code>blur</code> and submit) unless there is a good reason to do otherwise (for example, each time a user types a new letter).</p>
```

Comment:
<p>It is recommended that JavaScript-based validation be made in the <code>onchange</code> event (native to <code>select</code>, <code>input[type=checkbox]</code>, and <code>input[type=radio]</code>, also fired by some frameworks on text <code>blur</code> and submit). Your designer can advise on exceptions to this recommendation if there is a good reason to do otherwise (for example, each time a user types a new letter).</p>